### PR TITLE
⬆️ UPDATE: myst-parser~=0.13.1

### PR DESCRIPTION
**Pull request by chrisjsewell**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -74,10 +74,10 @@
             "numpy",
             "sympy",
             "pandas",
-            "jupytext>=1.6.0",
+            "jupytext-ebp==1.7.2a0",  # TODO waiting for jupytext 1.7.2 release
         ],
         "rtd": [
-            "jupytext>=1.6.0",
+            "jupytext-ebp==1.7.2a0",  # TODO waiting for jupytext 1.7.2 release
```
Comment (choldgraf) [inline setup.py:80]: I think this is now out but it is 1.8.0: https://github.com/mwouts/jupytext/releases/tag/v1.8.0

**Review by choldgraf — COMMENTED** [review]
Sorry for the slow turnaround, the holidays got my to-do list all messed up. This looks good to me w/ one comment about the Jupytext version. I'm not sure why there are suddenly errors popping up on the execution tests though :-/

https://github.com/executablebooks/MyST-NB/pull/284/checks?check_run_id=1586245248#step:5:36

File: setup.py (hunk 1 above)
Comment (chrisjsewell) [reply setup.py:80]: Oh yeh I know 👍 https://github.com/mwouts/jupytext/pull/692

**Comment by choldgraf** [comment]
🎉

**Pull request merged**